src/plugins/blocksmith/components/article-block-modal/CustomizeOptions.jsx

```diff
@@ -68,6 +71,11 @@ const CustomizeOptions = ({
         <Checkbox size="sd" id="hasUpdatedDate" checked={hasUpdatedDate} onChange={handleChangeCheckbox}>
           Is update
         </Checkbox>
+        <Checkbox size="sd" id="aside" checked={aside} onChange={handleChangeCheckbox} className="ml-3">
+          Aside
+        </Checkbox>
+        <Icon imgSrc="info-outline" id="aside-tooltip" size="xs" style={{ marginLeft: '0.3rem' }} />
```

Comment:
@nanetteCodes could the inline style here be accomplished with a class? eg `ml-2` adds some margin on the left